examples/bun/README.md

```diff
@@ -0,0 +1,14 @@
+- [Bun](https://bun.sh)
+- Vanilla TRPCClient in Deno
+
+[Install Bun](https://bun.sh/docs/installation) then run:
+
+```sh
+bun run src/server.ts
```

Comment:
```suggestion
bun run src/index.ts
```

Reply:
Or maybe rely on `bun run` being able to call `package.json` scripts directly with `bun start:server` and `bun start:client`?

Reply:
yep that's better